lib/coverband/adapters/memcached_store.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+module Coverband
+  module Adapters
+    class MemcachedStore < Base
+      STORAGE_FORMAT_VERSION = 'coverband_3_2'
+
+      attr_reader :memcached_namespace
+
+      def initialize(memcached, opts = {})
+        super()
+        @memcached = memcached
+        @memcached_namespace = opts[:memcached_namespace]
```

Comment:
note: for me in the future to move all the various adapters to `:coverband_namespace` opposed to memcache_ and redis_namespace